Document AWS metric name filtering (#38023)
DataDog/documentation: content/en/getting_started/integrations/aws.md

Use the {{< ui >}}Metric Collection{{< /ui >}} tab on the [AWS integration page][8] to configure which services the Datadog integration collects metrics from.

### Filter metrics by metric name

> Use the {{< ui >}}Metric Collection{{< /ui >}} tab on the [AWS integration page][8] to filter the CloudWatch metrics that Datadog collects from each namespace. Expand a namespace in the CloudWatch metric collection table and choose an **Include** or **Exclude** filter:

Reviewer: ```suggestion
Use the {{< ui >}}Metric Collection{{< /ui >}} tab on the [AWS integration page][8] to filter CloudWatch metrics by namespace. Expand a namespace in the CloudWatch metric collection table and choose an **Include** or **Exclude** filter:
```

- **Include**: Collect only Datadog metric names that match the configured patterns for that namespace.
- **Exclude**: Collect all Datadog metric names for that namespace except those that match the configured patterns.

Each namespace can use one filter mode at a time. Filter patterns support lowercase letters, numbers, `.`, `_`, and `*`. For example, `aws.ec2.network_*` matches EC2 network metrics. The table previews how many metrics match each pattern before you save changes.

Metric name filters apply per namespace and are evaluated after the namespace is enabled for metric collection.

#### Required metrics

> Metric name filters cannot remove `aws.ec2.cpuutilization` or `aws.lambda.invocations`. Datadog always collects these metrics because they are used for billing.

Reviewer 2: I would rephrase that but I don't know how. I'm not sure we should expose how billing is computed in our public docs

Author: I agree but Matt thought it would be a good idea. I was thinking something more along the lines of 
>Metric name filters cannot remove `aws.ec2.cpuutilization` or `aws.lambda.invocations`. Datadog always collects these foundational/critical metrics.

Something along these lines. I'll wait for the docs folks to chime in.

Reviewer: I recommend removing the subheader and making this an alert. I added my suggestion below. If customers need to know about billing, we can add it back in.
